ipam/paxos/paxos.go

```diff
@@ -0,0 +1,254 @@
+package paxos
```

Comment:
There are a small number of non-`camel_case` identifiers in this package, which should be turned into `camelCase`